FEMO/Space_board/GameScene.swift

```diff
@@ -52,6 +52,11 @@ class GameScene: SKScene {
     override func didMove(to view: SKView) {
         setupTiles()
         createPlayer1()
+        let menu_buttonNode = self.childNode(withName: "Menu_button") as! SKSpriteNode
```

Comment:
We may want to declare the button at the top of the file as we did with the other nodes

Reply:
completed